pkgs/by-name/no/node-core-utils/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildNpmPackage rec {
+  pname = "node-core-utils";
+  version = "5.14.0";
+
+  src = fetchFromGitHub {
+    owner = "nodejs";
+    repo = "node-core-utils";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
This is the preferred way to fetch tagged releases.